File: PresentationApp/WebApi/DataModel/Project.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.DataModel
{
    public class Project
    {
        public int ProjectId { get; set; }
        public string Title { get; set; }
        public string Photo { get; set; }
        public string Description { get; set; }
        public int Position { get; set; }

        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }


        public string UserId { get; set; }
        public User User { get; set; }
        

        public ICollection<Info> Infos { get; set; }
    }
}

File: ClientApp/presentation-app/src/app/shared/project.ts
import {Info} from "./info";

export class Project {
    projectId: number;
    title: string;
    photo: string;
    description: string;
    position: number;
    startDate: Date;
    endDate: Date;

    userId: string;
    infos: Array<Info> = new Array<Info>();
}
